app/components/history-item.js

```diff
@@ -1,4 +1,4 @@
-import { htmlSafe } from '@ember/string';
+// import { htmlSafe } from '@ember/string';
```

Comment:
Looks like you forgot to remove this line.

Reply:
Thanks.